Missing dependency

I tried building etcd-ca with the latest golang Docker images and got the following error: gopath/src/github.com/coreos/etcd-ca/cmd/util.go:11:2: code in directory /go/etcd-ca/gopath/src/github.com/coreos/etcd-ca/Godeps/_workspace/src/golang.org/x/crypto/ssh/terminal expects import "golang.org/x/crypto/ssh/terminal"

@yichengq I just replicated your session on my box and the error is the same.

```
fourcube@devbox:/tmp$ mkdir gopath
fourcube@devbox:/tmp$ export GOPATH=/tmp/gopath
fourcube@devbox:/tmp$ cd gopath
fourcube@devbox:/tmp/gopath$ go get github.com/coreos/etcd-ca
src/github.com/coreos/etcd-ca/cmd/util.go:11:2: code in directory /tmp/gopath/src/github.com/coreos/etcd-ca/Godeps/_workspace/src/golang.org/x/crypto/ssh/terminal expects import "golang.org/x/crypto/ssh/terminal"
```

I'm using the following go version.
```
fourcube@devbox:~$ go version
go version go1.4.1 linux/amd64
```

--- a/Godeps/_workspace/src/golang.org/x/crypto/ssh/terminal/util.go
+++ b/Godeps/_workspace/src/golang.org/x/crypto/ssh/terminal/util.go
@@ -14,7 +14,7 @@
 // 	        panic(err)
 // 	}
 // 	defer terminal.Restore(0, oldState)
-package terminal // import "golang.org/x/crypto/ssh/terminal"
+package terminal // import "github.com/coreos/etcd-ca/Godeps/_workspace/src/golang.org/x/crypto/ssh/terminal"
 
 import (
 	"io"

